sihl-email/src/sihl_email.ml

```diff
@@ -144,14 +136,15 @@ module MakeSmtp (Config : SmtpConfig) : Sihl.Contract.Email.Sig = struct
       | Some html -> Letters.Html html
       | None -> Letters.Plain email.text
     in
-    let* sender = Config.sender ()
-    and* username = Config.username ()
-    and* password = Config.password ()
-    and* hostname = Config.hostname ()
-    and* port = Config.port ()
-    and* with_starttls = Config.start_tls ()
-    and* ca_path = Config.ca_path ()
-    and* ca_cert = Config.ca_cert () in
+    let* config = Config.config () in
```

Comment:
@aronerben If the module is already called `*Config` maybe we can call the function `fetch` or something like that? So the usage would be `Config.fetch()`.

Reply:
Yea you're right, adjusted!